x-pack/legacy/plugins/integrations_manager/common/types.ts

```diff
@@ -49,15 +49,17 @@ export interface RegistryPackage {
 // the public HTTP response types
 export type IntegrationList = IntegrationListItem[];
 
-export type IntegrationListItem = Installed<RegistryListItem> | NotInstalled<RegistryListItem>;
+export type IntegrationListItem = Installable<RegistryListItem>;
```

Comment:
So you don’t have to do the `|` each time. Went with `Installable` over `Maybe*`, `Either*`, etc.